package-lock.json

```diff
@@ -18,15 +18,15 @@
         "fuzzer": "^0.2.1",
         "gh-pages": "^2.0.1",
         "intl-pluralrules": "^1.0.0",
-        "mocha": "^8.3.2",
+        "mocha": "^9.0.2",
         "nyc": "^15.1.0",
         "prettyjson": "^1.2.1",
         "rollup": "^1.9.3",
         "typedoc": "^0.20.34",
         "typescript": "^4.2.3"
       },
       "engines": {
-        "node": ">=10.0.0"
+        "node": ">=12.0.0"
```

Comment:
Thought: I double checked that 12 is the current LTS, so this change looks good to me.

Reply:
Yeah, this is a result of the changes in #557 which weren't completely reflected in the lockfiles.